src/RenderWebGL.js

```diff
@@ -621,6 +621,60 @@ RenderWebGL.prototype._drawThese = function(
 };
 
 /**
+ * Draw say bubble next to the Drawable
+ * @param {int} drawableID A Drawable ID for draw bubble to
+ * @param {string} text A text for render
+ */
+
+RenderWebGL.prototype.renderBubble = function (drawableID, text) {
+    var  drawable = Drawable.getDrawableByID(drawableID)
+        ,bounds = drawable.getBounds()
+        ,xy = {x: bounds.right + 5,
```

Comment:
This style of `,variable=...` doesn't exist anywhere in our codebase. Define the variable with separate `var`s instead?